benchopt/cli/helpers.py

```diff
@@ -40,6 +47,247 @@ def clean(benchmark, token=None, filename=None):
     rm_folder(cache_folder)
 
 
+def check_conda_env(env_name, benchmark_name=None):
```

Comment:
I would put this function in `benchopt.utils.conda_cmd` maybe as it is strongly linked to `conda`?

Reply:
Also, can it be used for `run` and `install` in `benchopt.cli.main`?

Reply:
I will see how it is possible to make it usable for `run` and `install` in `benchopt.cli.main` since they both manage the `recreate` option and `info` not.

Reply:
Oh I did not think about that. Let's merge this PR and do this move/refactor in another one!